### DISCUSSION TITLE
能否为「直播首页静音」增加或改为 暂停播放

### DISCUSSION THREAD

**Discussion by YeJay99** [the1812/Bilibili-Evolved#4772] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/issues/4459#issuecomment-1817775578 现在已经默认静音了，但是在隐藏板块后依然会占用网络、cpu和gpu，希望可以暂停播放

**Comment by the1812** [comment]
这个组件不维护了
